### ISSUE TITLE
ASP.NET Core Blazor Language Services for Mac?

### ISSUE THREAD

**Issue by hcientist** [aspnet/Blazor#336]
I think [that](https://developercommunity.visualstudio.com/content/problem/67655/cant-install-vsix-to-vs-for-mac.html) means I need an mpack instead of a vsix?

**Comment by danroth27** [comment]
Yup, we don't have tooling support for VS for Mac yet. It's something we plan to look into though, especially now they have Razor tooling support in preview.

**Comment by danroth27** [comment]
We're going to take a look at this and see how far we can get with the latest VS for Mac preview.

**Comment by danroth27** [comment]
Didn't make it for 0.2.0. Moving to next milestone

**Comment by vertonghenb** [comment]
Any help required with this or are we waiting for the Visual Studio for Mac team?

**Comment by danroth27** [comment]
No help required at this time. We're still working with the VS for Mac folks on this. No ETA at this time.

**Comment by stefandevo** [comment]
This does seem to work in Visual Code...

**Comment by hcientist** [comment]
sorry @stefandevo, just to be sure, are you on MacOS?

**Comment by stefandevo** [comment]
@hcientist yes, running the FlightFinder

**Comment by vertonghenb** [comment]
C# 7.3 for Visual Studio Mac will be supported soon :
https://developercommunity.visualstudio.com/content/problem/294400/support-c-73.html

**Comment by nathanielcook** [comment]
Looks like C# 7.3 on macOS is only partially implemented. Still not supported by the csc command. See https://github.com/mono/mono/issues/9796

**Comment by vertonghenb** [comment]
In the latest version of visual for Mac (preview) its now possible to run and build blazer apps!
Any update on the tooling yet? What is holding us back?

**Comment by danroth27** [comment]
@vertonghenb It's certainly something we want to do. It's just a matter of time and resources.

**Comment by vertonghenb** [comment]
@danroth27 
Well I'm just a bit confused since it's number 2 in the priority backlog. https://github.com/aspnet/Blazor/projects/3 

I know there are more important features out there, I would just like to know the roadmap and where we can place this issue in it. 

Not so long ago there was a demo if Razor tooling in VS Code on live.asp.net, do we intent to initially support vs code or is the road to support VS for Mac shorter? Can the community help out to tackle this?

**Comment by danroth27** [comment]
> I know there are more important features out there, I would just like to know the roadmap and where we can place this issue in it.

That's completely reasonable. The priority backlog is a bit stale right now. I'll make sure it gets updated.

> Not so long ago there was a demo if Razor tooling in VS Code on live.asp.net, do we intent to initially support vs code or is the road to support VS for Mac shorter? Can the community help out to tackle this?

We intend to support both. We had investigated supporting VS for Mac a while back and hit some blocking issues with the Razor tooling that we need to revisit. Currently we're investigating what it would take to get basic Razor tooling support in VSCode. While the foundational piece we need in VS for Mac are further along, we think there's probably more value in VSCode tooling as that's currently the tool of choice for lots of web developers.
